app/routers/dogfinder.py

```diff
@@ -25,48 +25,53 @@
 vecotrDBClient: IVectorDBClient
 
 dog_class_definition = {
-        "class": "Dog",
-        "invertedIndexConfig": {
-            "indexNullState": True,
-            "indexTimestamps": True
+    "class": "Dog",
+    "invertedIndexConfig": {
+        "indexNullState": True,
+        "indexTimestamps": True
+    },
+    "properties": [
+        {
+            "name": "breed",
+            "dataType": ["text"],
+            "description": "Name of dog breed"
         },
-        "properties": [
-            {
-                "name": "breed",
-                "dataType": ["text"],
-                "description": "Name of dog breed"
-            },
-            {
-                "name": "type",
-                "dataType": ["text"],
-                "description": "Found Or Lost"
-            },            
-            {
-                "name": "filename",
-                "dataType": ["text"],
-                "description": "Uploaded filename"
-            },
-            {
-                "name": "image",
-                "dataType": ["blob"],
-                "description": "Image"
-            }
-        ]
-    }
-
+        {
+            "name": "type",
+            "dataType": ["text"],
+            "description": "Found Or Lost"
+        },
+        {
+            "name": "filename",
+            "dataType": ["text"],
+            "description": "Uploaded filename"
+        },
+        {
+            "name": "image",
+            "dataType": ["blob"],
+            "description": "Image"
+        }
+    ]
+}
 
 
 # This is the request model for the add_documents endpoint
 class DocumentRequest(BaseModel):
     documents: List[DogDocument]
 
+
+is_found_field = "isFound"
```

Comment:
This should be uppercase as it is constant

Reply:
moved to upper case, used in the schema